CONTRIBUTING.md

```diff
@@ -80,62 +106,30 @@ By viewing the "Files changed", you can leave a comment on any line of the diff.
 Any discussion you have in a specific part of the diff will also be automatically hidden once a change is pushed that addresses it, which helps keep the pull request page clear and readable.
 
-When leaving line comments, make sure you leave them on the *diff* - not on the page for an individual commit.
-Any comments left on the commit will disappear from the pull request page as soon as that commit is no longer in the branch, even if a newer revision of that commit is.
-
 Anyone can participate in code review discussions.
 Feel free to jump in if you have something to contribute on another pull request, even if you're not the one who opened it.
 
 ### Cleaning up the commit history
 
 Once code review is finished or nearly finished, and no further development is planned on the branch, the branch's commit history should be cleaned up.
 You can alter the commit history of a branch using git's powerful [interactive rebase feature](http://www.git-scm.com/book/en/Git-Tools-Rewriting-History).
-The following few criteria help outline what makes for a clean commit history:
-
-#### Commits should be specific and atomic
-
-Any single commit should cover a specific change.
-That change might span multiple files, but it shouldn't introduce multiple distinct functional changes unless it's impossible to separate those changes from each other.
-It's okay (and normal!) for a branch that makes a lot of changes to consist of many small commits, each of which makes individual small changes.
-This makes it easier to revert individual commits from a branch, and also makes it easier to isolate the commit that caused a particular bug with git-blame.
-
-In development, it's common to introduce a change with a commit and then introduce multiple other commits with small fixes for that change.
-When getting ready for merging, those commits should all be squashed together.
 
-#### Every commit should work
-
-No individual commit should put the software in a broken state.
-It's fine for code to go unused (because the feature it's been introduced for isn't there yet), but no new functional issues should be introduced by a given commit.
-This ensures that reverting any individual commit from the branch is safe, and that git-bisect stays reliable for tracking down bugs.
-
-#### Commit summaries should be concise
-
-The commit summary (the first line of a commit message) should be short and clear.
-Usually, the first line of the commit message should be 50 - 80 characters.
-Since the summary is just a *summary* of the change, it should be readable at a glance when looking through git history.
-
-A good commit summary makes it clear a) what *part* of the code is changing, and b) what the change is.
-For example:
+### Small changes
 
-Clear commit summary:
-> normalize: fall back to default rule for unidentified files
+Small contributions such as fixing spelling errors, where the content is small enough to not be considered intellectual property, do not require signing a CLA.
```

Comment:
This is how we've been approaching things previously, inspired by Django's policy. Do we want to make it official?